common/log/zap_logger.go

```
@@ -239,10 +241,45 @@ func buildZapLogger(cfg Config, disableCaller bool) *zap.Logger {
 		ErrorOutputPaths: []string{outputPath},
 		DisableCaller:    disableCaller,
 	}
-	logger, _ := config.Build()
+	var logger *zap.Logger
+	if cfg.EnableRotation && len(cfg.OutputFile) > 0 {
+		core, _ := rotationZapCore(cfg, config)
```

We need to check and respond to this error, please don't swallow it